wire-ios-sync-engine/Source/UserSession/Search/SearchTask.swift

```diff
@@ -320,7 +320,7 @@ extension SearchTask {
     func performRemoteSearch() {
         guard
             let apiVersion = BackendInfo.apiVersion,
-            apiVersion >= .v2,
+            apiVersion >= .v1,
```

Comment:
actually available to v0, don't know if anyone is running v0 or the impact on this